cli/pcluster/configure/networking.py

```diff
@@ -219,20 +216,9 @@ def _get_availability_zone(aws_region_name):
     return ""
 
 
-def _evaluate_aws_region(aws_region_name):
-    if aws_region_name:
-        region = aws_region_name
-    elif os.environ.get("AWS_DEFAULT_REGION"):
-        region = os.environ.get("AWS_DEFAULT_REGION")
-    else:
-        region = DEFAULT_AWS_REGION_NAME
-    return region
-
-
 @handle_client_exception
-def ec2_conn(aws_region_name):
-    region = _evaluate_aws_region(aws_region_name)
-    ec2 = boto3.client("ec2", region_name=region)
+def ec2_conn():
```

Comment:
shall we get rid of this function now that it doesn't add any value?

Reply:
Done